.gitlab-ci-check-python3-format.yml

```diff
@@ -49,14 +49,12 @@ stages:
   - test
 
 test:check-python3-formatting:
-  image: debian
+  image: python:3.8.3
```

Comment:
Should we just set it to `python:3.8` so that we get bugfixes updates?

Reply:
good point! should now be `3.8` 👌